.github/tests-visual/visual-record.js

```diff
@@ -233,6 +221,31 @@ async function main() {
                 await createScreenshot(page, fileName);
             }
 
+            // test debug mode
```

Comment:
Ggf diese abschnitte in privat methoden verlagern?

Reply:
Ich persönlich finde es lesbarer, so wie es ist. Man hangelt sich durch die verschiedenen Kontexte, triggert irgendwas, wartet, stellt es wieder zurück, usw. — Ein Auslagern macht es nicht übersichtlicher, fände ich, eher im Gegenteil.